src/consensus.rs

```diff
@@ -50,11 +48,26 @@ pub async fn run(shutdown: impl Future) -> crate::Result<()> {
 impl Consensus {
     /// Run consensus
     async fn _run(&mut self) -> crate::Result<()> {
-        let (_tx, mut rx): (Sender<bool>, Receiver<bool>) = broadcast::channel(1);
+        let (saito_message_tx, mut saito_message_rx) = broadcast::channel(32);
```

Comment:
Would like to just call these sender and receiver rather than tx and rx.

Reply:
"Tx" and "rx" stand for "transmitter" and "receiver" and are the common nomenclature used with tokio channels. I would prefer to stick with convention on this one